statsd/container_stub.go

```diff
@@ -3,7 +3,7 @@
 
 package statsd
 
-func initContainerID(userProvidedID string, cgroupFallback bool) {
+var initContainerID = func(userProvidedID string, cgroupFallback bool) {
```

Comment:
It was necessary because `initContainerID` would be used in every test, breaking them in some env